front-end/src/app/shared/services/FecApiService/fec-api.service.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpClient } from '@angular/common/http';
+import { environment } from '../../../../environments/environment';
+import { Injectable } from '@angular/core';
+import { map, Observable } from 'rxjs';
+import { CommitteeAccount } from '../../models/committee-account.model';
+import { FecApiPaginatedResponse } from 'app/shared/models/fec-api.model';
+
```

Comment:
Would you move the fec-api.service.ts and fec-api.service.spec.ts up a directory into the shared/services directory? I left AuthService and SessionService in own directories because they will be removed after Login.gov set up. It's legacy code